cli/src/cluster_query.rs

```diff
@@ -1961,7 +1961,7 @@ pub fn process_wait_for_max_stake(
     max_stake_percent: f32,
 ) -> ProcessResult {
     let now = std::time::Instant::now();
-    rpc_client.wait_for_max_stake(config.commitment, max_stake_percent)?;
+    rpc_client.wait_for_max_stake(config.commitment, max_stake_percent, None)?;
```

Comment:
This path is only taken from the CLI. Seems fine to keep the existing behavior letting it wait forever